packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -901,12 +821,7 @@ contract VotingV2 is
      * @return pendingRequests array containing identifiers of type `PendingRequest`.
      * and timestamps for all pending requests.
      */
-    function getPendingRequests()
-        external
-        view
-        override(VotingV2Interface, VotingAncillaryInterface)
-        returns (PendingRequestAncillary[] memory)
-    {
+    function getPendingRequests() external view override(VotingV2Interface) returns (PendingRequestAncillary[] memory) {
```

Comment:
nit: I don't think you need to specify which contract you're overriding if you're only overriding one.

```suggestion
    function getPendingRequests() external view override returns (PendingRequestAncillary[] memory) {
```